js/classes/entities/midground/chad.js

```diff
@@ -105,14 +114,19 @@ class Chad {
     static get RUN_MULTIPLIER() {
         return 1.4;
     }
-    /** The mulitiplier that allows CHAD to run. */
+    /** How much faster Chad is while dashing than walking */
     static get DASH_MULTIPLIER() {
         return 3.5;
     }
 
     /** The barrier that limits the length of the longest dash. */
-    static get DASH_BARRIER() {
-        return 250;
+    static get DASH_LIMIT() {
+        return 280;
+    }
+
+    /** The delay between dashes in seconds. */
+    static get GROUND_DASH_COOLDOWN() {
```

Comment:
I like that we are using a cool down. It seems a little odd to me to use a cool down for the dash on the ground and the dash in the air if that is what is happening. Or is it a timer for ground and distance in air? Can we make the dash in the air and ground consistent by using a timer for both as we recently discussed?